tests/test_node.py

```diff
@@ -836,6 +839,96 @@ def render(self, context: Context, name: str, **kwargs) -> str:
 
         TestNode.unregister(component_tags.register)
 
+    def test_node_class_attributes(self):
```

Comment:
Added test to check also for other attributes on `BaseNode` as we don't check for that yet.